content/shmem_broadcast.tex

```diff
@@ -127,17 +140,24 @@
 constraints, which are as follows:
 }{Routine}{Data type of \VAR{dest} and \VAR{source}}
 
-\apitablerow{shmem\_broadcastmem}{\Cstd: Any data  type.  nelems is scaled in bytes.}
-\apitablerow{shmem\_broadcast64}{No \CorCpp{} structures are allowed.}
-\apitablerow{shmem\_broadcast32}{No \CorCpp{} structures are allowed.}
+\apitablerow{\FUNC{shmem\_broadcastmem}}{\source{} and \dest{} objects are
+  byte aligned; \VAR{nelems} is the object size in bytes}
+\apitablerow{\FUNC{shmem\_broadcast64}}{\source{} and \dest{} objects are
+  8-byte aligned; \VAR{nelems} is the count of the number of 8-byte
+  elements}
+\apitablerow{\FUNC{shmem\_broadcast32}}{\source{} and \dest{} objects are
+  4-byte aligned; \VAR{nelems} is the count of the number of 4-byte
+  elements}
```

Comment:
Are we ok with deleting the `apidesctable` (and below in collect) in the merge with #276?